### PR TITLE
Hongyli alert browser 4.5

### PR DESCRIPTION
**Pull request by lihongyan1**
Change code to make monitoring UI cases can work on 4.5 and previous releases
Depend on PR:
https://github.com/openshift/cucushift/pull/7665
The PR also fixed intermittent test failure when integration run
https://issues.redhat.com/browse/OCPQE-231
https://issues.redhat.com/browse/OCPQE-232
V3 job on 4.4: all UI cases pass
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/87917/
V3 job on 4.5: all UI cases pass
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/88145/console

### CODE REVIEW

**Comment by lihongyan1** [comment]
@juzhao @yapei Please help review

File: lib/rules/web/admin_console/4.1/monitoring_alerts.xyaml [hunk 1]
```diff
@@ -29,6 +29,10 @@ perform_action_item:
     button_text: <item_text>
   action: click_actions_button
   action: click_button_text
+perform_silence:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.1/monitoring_alerts.xyaml:32]: Does `submit_changes` meet your requirement?

Comment (juzhao) [reply lib/rules/web/admin_console/4.1/monitoring_alerts.xyaml:32]: perform_silence is fine, create_silence is fine too, the function name should be easy to understand and show what is the function do

Comment (juzhao) [reply lib/rules/web/admin_console/4.1/monitoring_alerts.xyaml:32]: no more comments from me

Comment (lihongyan1) [reply lib/rules/web/admin_console/4.1/monitoring_alerts.xyaml:32]: In 4.4 and previous release, the button named create. Frome 4.5, the button named Silence

Comment (lihongyan1) [reply lib/rules/web/admin_console/4.1/monitoring_alerts.xyaml:32]: changed code, action perform_silence will call action submit_changes
thx

**Pull request merged**